src/v2/Components/Authentication/Views/SignUpForm.tsx

```diff
@@ -301,6 +301,7 @@ export const SignUpFormQueryRenderer: React.FC<FormProps> = passedProps => {
           }
         }
       `}
+      placeholder={<SignUpForm {...passedProps} />}
```

Comment:
This fixes an issue where right now, when a user goes to the sign up page, it shows the title but there's a long delay while the query renderer is fetching geo location. For now, just pass in form; the rest will come by the time the user interacts